common/components/WalletDecrypt/DeterministicWalletsModal.jsx

```diff
@@ -40,13 +40,15 @@ type Props = {
   dPaths: { label: string, value: string }[],
   publicKey: string,
   chainCode: string,
+  seed: string,
```

Comment:
This is a required prop, but the DecryptWallet/Trezor.jsx wasn't altered to send it (And probably shouldn't, I think this should be optional.) Any idea why flow didn't catch that?

Reply:
Agree, I'll make it optional. Not sure why flow didn't catch that, it definitely should have...